templates/locations/bigmap.html

```diff
@@ -0,0 +1,156 @@
+{% extends "base.html" %}
+{% load i18n %}
+{% load map_tags %}
+
+{% block content %}
+
+<div class="container-fluid">
+    <div class="row-fluid">
+        <div class="col-xs-2 col-sm-2 col-md-2 col-lg-2">
+            {% for location in locations %}
+                <div class="panel panel-default">
+                    <div class="panel-heading">
+                        <a href="{{ location.get_absolute_url }}">{{ location.name }}</a>
+                    </div>
+                    <div class="panel-body">
+                        <p>{{ location.description }}</p>
+                    </div>
+                </div>
+            {% endfor %}
+
+            <div class="pagination">
+                <span class="step-links">
+                    {% if locations.has_previous %}
+                        <a href="?page={{ locations.previous_page_number }}">previous</a>
+                    {% endif %}
+
+                    <span class="current">
+                        Page {{ locations.number }} of {{ locations.paginator.num_pages }}.
+                    </span>
+
+                    {% if locations.has_next %}
+                        <a href="?page={{ locations.next_page_number }}">next</a>
+                    {% endif %}
+                </span>
+            </div>
+        </div>
+        <div class="col-xs-10 col-sm-10 col-md-10 col-lg-10">
+            <div>
+                {% map initLocation.latitude initLocation.longitude %}
```

Comment:
Adica, daca vrei sa nu depinda de o locatie centrul, mai bine pui logica in server